scripts/probe_xas_nrtidalv3_bbh_phase.py

```
@@ -0,0 +1,159 @@
+    ripple_phi = bbh_psi + phase_shift  # equivalent to LAL's phi(f) modulo conv.
+
+    # ----- Compare -----
+    # Caveat: there can be an overall constant phase difference due to phi0
+    # convention. Remove it; what we really want is to see if there's a
+    # nonlinear-in-f residual.
+    diff = ripple_phi - lal_phi
+    # remove const + linear-in-f best fit
+    A = np.vstack([fb, np.ones_like(fb)]).T
+    coef, *_ = np.linalg.lstsq(A, diff, rcond=None)
+    resid = diff - A @ coef
+    print(f"\nripple_phi - lal_phi (linear+const removed):")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)